test/e2e/operator_test.go

```diff
@@ -492,13 +498,20 @@ func TestMultipleContours(t *testing.T) {
 
 	// Ensure the default contour can be deleted and clean-up.
 	for _, testName := range testNames {
+		// Verify the user-defined namespace was removed by the operator.
+		ns := fmt.Sprintf("%s-ns", testName)
```

Comment:
@sunjayBhatia this is the same as https://github.com/projectcontour/contour-operator/pull/328/files?file-filters%5B%5D=.go&file-filters%5B%5D=No+extension#diff-cf4b4e5424070a666a364d1d1b04011478d888d6d3d65c943ca7783333b7a6e4R472. Can you define this once, say after L467 and use the var throughout?